dom2/character_data_test.cpp

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2021 Robin Lindén <dev@robinlinden.eu>
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "dom2/character_data.h"
+
+#include "etest/etest.h"
+
+#include <string>
+#include <string_view>
```

Comment:
Should `<cstddef>` and `<utility>` be added?

Reply:
You're right, they should be! :x `std::size_t` and `std::move`.